config/standard/targets.js

```diff
@@ -35,15 +44,15 @@ var targets = [
     subtitle_translation_key: 'targets.this_month.subtitle',
 
     appliesTo: 'reports',
-    appliesIf: isNewestPregnancy,
-    emitCustom: function(c, r) {
+    appliesIf: extras.isNewestPregnancy,
+    emitCustom: function(inst, c, r) {
```

Comment:
I don't see `inst` used... are you making this a new function template? Would be good to mention `emitCustom` in [this targets section](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#target-schema), with details there or at documented in a separate page on advanced techniques.

Reply:
Will document emitCustom. Looks like we don't have that documented at all...  It is a quite a weird one to document its nuances.